README.md

```diff
@@ -1284,6 +1284,14 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
 * [nut](https://github.com/jingweno/nut) - Vendor Go dependencies.
 * [VenGO](https://github.com/DamnWidget/VenGO) - create and manage exportable isolated go virtual environments.
 
+## Performance
+
+* [jaeger](https://github.com/jaegertracing/jaeger) - A distributed tracing system.
+* [opencensus](https://github.com/census-instrumentation/opencensus-go) - A stats collection and distributed tracing framework.
+* [opentracing](https://github.com/opentracing/opentracing-go) - OpenTracing API for Go.
```

Comment:
new packages:
- https://github.com/opentracing/opentracing-go
- https://github.com/census-instrumentation/opencensus-go
- https://github.com/jaegertracing/jaeger

pls read: https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standards

Reply:
yep, I will fix that